packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -78,9 +85,10 @@ async function fetchSubscribedProductsFromStripe(uid, stripeHelper, email) {
   const subscribedProducts = [
     // All accounts get this psuedo-product
     PRODUCT_REGISTERED,
-    ...customer.subscriptions.data.map(
-      ({ plan: { product: productId } }) => productId
-    ),
+    ...customer.subscriptions.data
+      // TODO: Centralize subscription filtering logic for active subs
```

Comment:
This might be worth an `activeSubscriptionsForCustomer` method in StripeHelper someday, but 🤷‍♂ this works for now

Reply:
Yea, mainly if Stripe add's more, as its pretty clear-cut at the moment.